chain/network/src/routing/graph/tests.rs

```diff
@@ -67,22 +67,22 @@ async fn one_edge() {
     let e1 = data::make_edge(&node_key, &p1, 1);
     let e1v2 = e1.remove_edge(peer_id(&p1), &p1);
 
-    tracing::info!(target:"test", "Add an active edge. Update RT with pruning.");
+    tracing::info!(target:"test", "Add an active edge. Update RT with pruning");
```

Comment:
```suggestion
    tracing::info!(target:"test", "add an active edge, update RT with pruning");
```

Reply:
I'll check why these weren't included in the automated changes....